src/css/common/visibility.css

```diff
@@ -0,0 +1,52 @@
+
```

Comment:
This whole concept of `visible-xxx` and `hidden-xxx` classes seems a bit unusual to me. Is it some common approach? Could you direct me to some articles about it?